src/main/java/nl/novi/serviceconnect/dtos/TransactionInputDto.java

```diff
@@ -1,14 +1,16 @@
 package nl.novi.serviceconnect.dtos;
 
-import java.io.File;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.Date;
 
 public class TransactionInputDto {
     private Date transactionDate;
+    @JsonProperty("isPayed")
     private boolean isPayed;
     private Long serviceRequestId;
 
-    public File invoice;
+    public String invoice;
```

Comment:
Het hoort niet mogelijk te zijn voor de gebruiker om een invoice mee te geven bij het updaten of aanmaken van een Transaction. De invoice wordt alleen meegeven bij een response/outputDto omdat deze gegenereerd wordt op de server.

Reply:
Deleted done,  dank je luuk